jenkins/BUILD

```diff
@@ -36,6 +33,7 @@ filegroup(
     name = "all-srcs",
     srcs = [
         ":package-srcs",
+        "//jenkins/aws-janitor:all-srcs",
```

Comment:
This does not seem right

Reply:
What checked-in code uses the AWS libraries?

Reply:
@fejta https://github.com/kubernetes/test-infra/blob/d611e670d939a7841e780b96233d6ae12d399b41/jenkins/aws-janitor/main.go#L29-L35

Reply:
what's not right about it? the all-srcs rule is supposed to include subpackages, which is what it's doing here.

Reply:
It should depend on the compiled binary as a data dep, not toss in the sources.

Interesting! How did that get checked in with neither unit tests nor a BUILD file?

Reply:
this is the `all-srcs` filegroup. It doesn't depend on any binaries.

and as noted in the PR description, we'd previously excluded the aws-janitor from bazel for (no-longer-valid) reasons.

Reply:
(the aws-janitor predates bazel)

Reply:
oh, my comment about why we'd excluded aws-janitor was actually in https://github.com/kubernetes/test-infra/pull/6201#issuecomment-356391825.

Reply:
This is not an appropriate way to fix this

Reply:
Nope nevermind, I'm totally wrong